src/schema/schema_create.c

```diff
@@ -9,45 +9,21 @@
 #include "wt_internal.h"
 
 /*
- * __wt_direct_io_size_check --
- *     Return a size from the configuration, complaining if it's insufficient for direct I/O.
```

Comment:
Since this function now only returns a size, and doesn't have other checks, it is not necessary anymore. 

Do we want to keep it for readability / tidyness purposes on should I default back to using __wt_config_gets and remove this function?

Reply:
removed here [f01f96a](https://github.com/wiredtiger/wiredtiger/pull/10430/commits/f01f96ade1e58c03d448ad659ae36610d7701c71)